dist/preview release/what's new.md

```diff
@@ -1,238 +1,241 @@
-# 5.0.0
```

Comment:
Would you be able to revert the changes to this file and make sure only your line changes?